src/test/java/com/shopify/ShopifySdkTest.java

```diff
@@ -421,6 +422,39 @@ public void givenSomePageAndCreatedAtMinAndCreatedAtMaxOrdersWhenRetrievingOrder
 
 		shopifyRefund1.setRefundLineItems(Arrays.asList(shopifyRefundedLineItem));
 
+		final ShopifyTransaction shopifyTransaction1 = new ShopifyTransaction();
+		shopifyTransaction1.setId("123");
+		shopifyTransaction1.setMessage("Refunded 12.72 from manual gateway");
+		shopifyTransaction1.setAmount(new BigDecimal(12.72));
+		shopifyTransaction1.setStatus("SUCCESS");
+		shopifyTransaction1.setKind("refund_discrepancy");
+		shopifyTransaction1.setGateway("manual");
+		shopifyTransaction1.setCurrency(Currency.getInstance("USD"));
+		shopifyTransaction1.setMaximumRefundable(new BigDecimal(15.99));
+
+		final ShopifyTransaction shopifyTransaction2 = new ShopifyTransaction();
+		shopifyTransaction1.setId("456");
```

Comment:
update this instead to be **shopifyTransaction2**.setId("456"). You are overwriting the **shopifyTransaction1** object here. Shouldn't have to change assertions because you are just checking at the index but just to ensure you are asserting on some values instead of null === null.